tests/vector_tools/interpolate.output

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Please also rename this file to `tests/vector_tools/interpolate.with_p4est=true.mpirun=1.output`. This test requires `MPI` and `p4est` and would fail if executed withput it.

Reply:
The file has been renamed!